journal/modernized/07-chapter-7.adoc

```diff
@@ -202,7 +202,7 @@ and one of them had been clerk to a public worship house;
 the opportunity closed under a solemn covering.
 
 Fifth Month 9th.--I came home last evening from the Yearly Meeting in Dublin,
-our dear friend, Sarah Grubb, with her companion, W. Allen, were there,
+our dear friend, S. G., with his companion, W. Allen, were there,
```

Comment:
Good catch here, this is Stephen Grellet, not Sarah Grubb. At some point a few of these `S. G.`'s got incorrectly changed to Sarah Grubb, including here.  I'm going to submit a separate PR to clean the rest of them up.